src/services/galleryService.js

```
@@ -0,0 +1,359 @@
+
+// --- CATEGORY CONFIG ---
+export const getCategoryConfig = () => {
+    try {
+        const stored = localStorage.getItem(CATEGORY_STORAGE_KEY);
+        return stored ? JSON.parse(stored) : {
+            order: [],
+            renames: {},
+            hidden: [],
+            custom: [] // { id: 'unique_id', name: 'DisplayName' }
+        };
+    } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)